src/components/ContactForm.js

```diff
@@ -1,35 +1,39 @@
 import styles from "@/styles/ContactForm.module.scss"
+import { useRouter } from "next/router"
+import lt from '@/locales/lt'
+import en from '@/locales/en'
+
 
 function ContactForm(props) {
+   const router = useRouter();
+   const t = router.locale === 'lt' ? lt : en
+
    
    return (
       <div className={styles.formTitleBlock}>
          {props.formTitle && 
          <div>
-            <h3>Your opinion is important to us</h3>
-            <p>Our vision is to create a world where everyone has access to financial 
-               services, enabling them to build a brighter future for themselves and their
-               communities. 
-            </p>
+            <h3>{t.inputForm.title}</h3>
+            <p>{t.inputForm.description}</p>
          </div>
          }  
          <form className={styles.formBlock}>
             <div className={styles.formInputBlock}>
             <input type="text" 
-            placeholder="Topic"
+            placeholder={t.inputForm.topic}
```

Comment:
please call it t.contactForm.topicPlaceholder

Reply:
Done to all inputs Placeholder's